localisation/00_wc_sad.csv

```diff
@@ -715,21 +715,21 @@ whispers_tooltip_tip_4;The Old Gods whisper myths of long forgotten times, a ble
 twilight_architecture;Twilight Architecture;;Zwielicht-Architektur;;;;;;;;;;;x
 twilight_architecture_desc;This character uses the powers of the Old Gods to use elementium as a building material.;;Dieser Charakter nutzt die Kr�fte der alten G�tter, um Elementium als Baumaterial zu verwenden.;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
-EVTDESC_WCUTF.1_loken;You have stayed in solitary reclusion ever since you accidently killed your lover and your brother's wife, Sif. The guilt and the whispers of Yogg-Saron gnaw at your thoughts, whittling away your mental fortitude and ability in being his jailer. Time will tell if you can redeem yourself or if falling into madness is your fate.;;Seit Ihr Eure Geliebte und Frau Eures Bruders Thorim, Sif, ermordet habt, lebt Ihr in Einsamkeit. Die Schuld an Eurer Tat und die Einfl�sterungen von Yogg-Saron nagen an jedem Eurer Gedanken und halten Euch unf�hig, den alten Gott wirksam zu b�ndigen und zu bewachen. Nur die Zeit wird zeigen, ob Ihr in der Lage seid, Euch selbst zu erl�sen, oder ob Ihr dem Wahnsinn verfallen werdet.;;;;;;;;;;;x
-EVTDESC_WCUTF.1_thorim;You have stayed in solitary reclusion ever since the murder of your wife Sif by the traitorous frost giant king Arngrim. Now you sit on your throne without motivation to continue your Keeper duties. There is no point in life without Sif.;;Seit der Ermordung Eurer Gattin Sif durch die verr�terischen Frostriesen lebt Ihr in Einsamkeit. Jetzt sitzt lhr auf Eurem Thron, ohne den Drang, Eure H�terpflichten fortzusetzen. Es gibt keinen Sinn in Eurem Leben ohne Sif.;;;;;;;;;;;x
+EVTDESC_WCUTF.1_loken;You have stayed in solitary reclusion ever since you accidently killed your lover and your brother's wife, Sif. The guilt and the whispers of Yogg-Saron gnaw at your thoughts, whittling away your mental fortitude and ability in being his jailer. Time will tell if you can redeem yourself or if falling into madness is your fate.;;Seit Ihr Eure Geliebte und Frau Eures Bruders Thorim, Sif, versehentlich get�tet habt, lebt Ihr in Einsamkeit. Die Schuld an Eurer Tat und die Einfl�sterungen von Yogg-Saron nagen an Euren Gedanken und halten Euch unf�hig, den alten Gott wirksam zu b�ndigen und zu bewachen. Nur die Zeit wird zeigen, ob Ihr in der Lage seid, Euch selbst zu erl�sen, oder ob Ihr dem Wahnsinn verfallen werdet.;;;;;;;;;;;x
+EVTDESC_WCUTF.1_thorim;You have stayed in solitary reclusion ever since the murder of your wife Sif by the traitorous frost giant king Arngrim. Now you sit on your throne without motivation to continue your Keeper duties. There is no point in life without Sif.;;Seit der Ermordung Eurer Gattin Sif durch den verr�terischen Frostriesenk�nig Arngrim lebt Ihr in Einsamkeit. Jetzt sitzt lhr auf Eurem Thron, entmutigt, Euren H�terpflichten noch weiter nachzugehen. Ohne Sif hat Euer Leben keinen Sinn.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCUTF.1_thorim;You stayed in solitary reclusion ever since the murder of your wife Sif by the traitorous King of the Frost Giants, Arngrim. Now you sit on your throne without motivation to continue your Keeper duties. There is no point in life without Sif.;;Seit der Ermordung Eurer Gattin Sif durch den verr�terischen Frostriesenk�nig Arngrim lebt Ihr in Einsamkeit. Jetzt sitzt lhr auf Eurem Thron, entmutigt, Euren H�terpflichten noch weiter nachzugehen. Ohne Sif hat Euer Leben keinen Sinn.;;;;;;;;;;;x
```
Rewrote the King part to better flow.